### PR TITLE
[javascript/cz] improve wording, fix typos, add missing text from English original

### LINKED ISSUE #2707: [javascript/cs] some recommendation in czech language
Hello,
first of all: email martinek@ludis.me   doesn't work.
// the rest is in czech lang.

Dobry den,
mel bych par navrhu jak upravit https://learnxinyminutes.com/docs/cs-cz/javascript/
original  je predpokladam zde: https://learnxinyminutes.com/docs/javascript/

1)
// Vyrazu můžou být spuštěny pomocí ;
delejNeco();

original:
// Statements can be terminated by ;
doStuff();

navrh:
// Příkazy mohou být ukončeny středníkem ;
doStuff();

2)
"Hello world"  bych neprekladal, je to priklad, ktery je vseobecne znamy. A vyhledavace ho najdou, coz prispeje k rozsireni clanku/stranky.

3)

// Existují také typy `null` a `undefined`.
null;       // značí, že žádnou hodnotu
undefined;  // značí, že hodnota nebyla definovaná (ikdyž
           // `undefined` je hodnota sama o sobě)

orig:
// There's also `null` and `undefined`.
null;       // used to indicate a deliberate non-value
undefined;  // used to indicate a value is not currently present (although
           // `undefined` is actually a value itself)

navrh:
// Existují také `null` a `undefined`.
null;       // obvykle označuje něco záměrně bez hodnoty       ###### slovo záměrně mi prijde dulezite
undefined;  //obvykle označuje, že hodnota není momentálně definovaná (ačkoli
           // `undefined` je hodnota sama o sobě)

4)
var  mojePole  =  ["Ahoj",  45,  true];

navrh:
var  myArray  =  ["Ahoj",  45,  true];        ##### viz dalsi radky, kde je ponechano myArray

5) nektere veci bych neprekladal

// `if` (když) funguje, jak byste čekali.

navrh:
// `if` funguje, jak byste čekali.

6) chybi
// Breaking out of labeled loops is similar to Java
outer:
for  (var  i  =  0;  i  <  10;  i++)  {
    for  (var  j  =  0;  j  <  10;  j++)  {
        if  (i  ==  5  &&  j  ==5)  {
            break  outer;
            // breaks out of outer loop instead of only the inner one
        }
    }
}

navrh:
// Opuštění cyklu s návěštím je podobné jako v Javě
outer:
for  (var  i  =  0;  i  <  10;  i++)  {
    for  (var  j  =  0;  j  <  10;  j++)  {
        if  (i  ==  5  &&  j  ==5)  {
            break  outer;
            // opustí vnější (outer) cyklus namísto pouze vnitřního (inner) cyklu
        }
    }
}

7) preklep
ma byt:  // 4. Funkce,

S pranim krasneho dne,
Martin

### PR DESCRIPTION
**Pull request by tomkuba**
Fixes #2707
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: cs-cz/javascript.html.markdown [hunk 1]
```diff
@@ -9,32 +9,27 @@ lang: cs-cz
 filename: javascript-cz.js
 ---
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Wouldn't you like to add yourself to the translators list?

File: cs-cz/javascript.html.markdown [hunk 2]
```diff
@@ -9,32 +9,27 @@ lang: cs-cz
...
-Google Chrome se stává více a více oblíbený pro serverovou část webových aplikací.
-
-Zpětná vazba je velmi ceněná. Autora článku můžete kontaktovat (anglicky) na
-[@adambrenecki](https://twitter.com/adambrenecki), nebo
-[adam@brenecki.id.au](mailto:adam@brenecki.id.au), nebo mě, jakožto překladatele,
-na [martinek@ludis.me](mailto:martinek@ludis.me).
+Google Chrome se stává stále oblíbenější i pro serverovou část webových 
+aplikací.
 
 ```js
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Are you sure that block needs to be deleted as a whole in Czech?

File: cs-cz/javascript.html.markdown [hunk 3]
```diff
@@ -65,30 +60,30 @@ delejNeco()
...
-// Také existují hodnoty typu bool
+// Také existují hodnoty typu boolean.
 true; // pravda
 false; // nepravda
 
-// Řetězce znaků jsou obaleny  ' nebo ".
+// Řetězce znaků jsou obaleny ' nebo ".
 'abc';
-"Ahoj světe!";
+"Hello, world";
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Why so? It seems, JS supports input in all languages.

File: cs-cz/javascript.html.markdown [hunk 4]
```diff
@@ -103,16 +98,16 @@ false; // nepravda
 2 >= 2; // = true
 
 // Řetězce znaků se spojují pomocí +
-"Ahoj " + "světe!"; // = "Ahoj světe!"
+"Hello " + "world!"; // = "Hello world!"
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Ditto, see previous comment

File: cs-cz/javascript.html.markdown [hunk 5]
```diff
@@ -103,16 +98,16 @@ false; // nepravda
...
 
-// ... což funguje nejenom s řetězci
+// ... což funguje nejen s řetězci
 "1, 2, " + 3; // = "1, 2, 3"
-"Ahoj " + ["světe", "!"] // = "Ahoj světe,!"
+"Hello " + ["world", "!"]; // = "Hello world,!"
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Ditto

File: cs-cz/javascript.html.markdown [hunk 6]
```diff
@@ -124,24 +119,24 @@ null === undefined; // = false
...
 "13" + !0; // '13true'
 
-// Můžeme přistupovat k jednotlivým znakům v řetězci pomocí charAt`
+// Můžeme přistupovat k jednotlivým znakům v řetězci pomocí `charAt`
 "Toto je řetězec".charAt(0);  // = 'T'
 
-// ...nebo použít `substring` k získání podřetězce
-"Ahoj světe".substring(0, 4); // = "Ahoj"
+// ...nebo použít `substring` k získání podřetězce.
+"Hello world".substring(0, 5); // = "Hello"
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Ditto

File: cs-cz/javascript.html.markdown [hunk 7]
```diff
@@ -124,24 +119,24 @@ null === undefined; // = false
...
 
-// `length` znamená délka a je to vlastnost, takže nepoužívejte ()
-"Ahoj".length; // = 4
+// `length` znamená délka a je to vlastnost, takže nepoužívejte ().
+"Hello".length; // = 5
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: Ditto

File: cs-cz/javascript.html.markdown [hunk 8]
```diff
@@ -124,24 +119,24 @@ null === undefined; // = false
...
 
 // Existují také typy `null` a `undefined`.
-null;      // značí, že žádnou hodnotu
-undefined; // značí, že hodnota nebyla definovaná (ikdyž
+null;      // obvykle označuje něco záměrně bez hodnoty
+undefined; // obvykle označuje, že hodnota není momentálně definovaná (ačkoli
            // `undefined` je hodnota sama o sobě)
 
 // false, null, undefined, NaN, 0 and "" vrací nepravdu (false). Všechno ostatní
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: ...0 a "" vrací nepravdu

File: cs-cz/javascript.html.markdown [hunk 9]
```diff
@@ -474,21 +493,41 @@ mujPrototyp.__proto__ = {
...
 
-// Zde neni žádné kopírování; každý objekt ukládá referenci na svůj prototyp
-// Toto znamená, že můžeme měnit prototyp a změny se projeví všude
+// Zde není žádné kopírování; každý objekt ukládá referenci na svůj prototyp
+// Toto znamená, že můžeme měnit prototyp a změny se projeví všude.
 mujPrototyp.smyslZivota = 43;
-mujObjekt.smyslZivota // = 43
+mujObjekt.smyslZivota; // = 43
+
+// Příkaz for/in umožňuje iterovat vlastnosti objetku až do úrovně null
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: objektu?

File: cs-cz/javascript.html.markdown [hunk 10]
```diff
@@ -425,46 +440,50 @@ Math.min.apply(Math, [42, 6, 27]); // = 6
...
 mujObjekt = new MujKonstruktor(); // = {mojeCislo: 5}
 mujObjekt.mojeCislo; // = 5
 
-// Každý JsavaScriptový objekt má prototyp. Když budete přistupovat k vlasnosti
-// objektu, který neexistuje na objektu, tak se JS koukne do prototypu.
+// Narozdíl od nejznámějších objektově orientovaných jazyků, JavaScript nezná
+// koncept instancí vyvořených z tříd. Místo toho Javascript kombinuje
+//  instanciování a dědění do konceptu zvaného 'prototyp'.
+
+// Každý JsavaScriptový objekt má prototyp. Když budete přistupovat k vlastnosti
```
Comment (Menelion) [inline cs-cz/javascript.html.markdown]: JavaScriptový

File: cs-cz/javascript.html.markdown (hunk 1 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: No, I am OK with just being in git commit history.

File: cs-cz/javascript.html.markdown (hunk 2 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: The block is not in source text. Also this duplicates text as author is also mentioned in footer. ludis.me domain does not exist for more than a year. I think we can remove the mail address. Author still remains mentioned in translators section.

File: cs-cz/javascript.html.markdown (hunk 3 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: `Hello world` is not usually being translated to Czech as it is generally know term. It is used in tutorials for beginners in programming. I think that this text targets audience with at least some experience. So I think `Hello world` fits better.

File: cs-cz/javascript.html.markdown (hunk 10 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: fixed

File: cs-cz/javascript.html.markdown (hunk 9 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: fixed

File: cs-cz/javascript.html.markdown (hunk 8 above)
Comment (tomkuba) [reply cs-cz/javascript.html.markdown]: fixed

**Comment by Menelion** [comment]
Thanks!

**Pull request merged**